tests/src/model/pages/run-image-page.ts

```diff
@@ -81,22 +80,7 @@ export class RunImagePage extends BasePage {
       throw Error(`Start Button not enabled: ${errMessage}`);
     }
     await this.startContainerButton.click();
-    // wait for containers page to appear
-    const containers = new ContainersPage(this.page);
-    try {
-      await waitWhile(
-        async () => await this.startContainerButton.isVisible(),
-        5000,
-        900,
-        true,
-        'Error starting a container',
-      );
-    } catch (err) {
-      await this.errorAlert.waitFor({ state: 'visible', timeout: 2000 });
-      const errMessage = await this.errorAlert.innerText({ timeout: 1000 });
-      throw new Error(`Error starting a container: ${errMessage}`);
-    }
-    return containers;
+    return new ContainersPage(this.page);
```

Comment:
Same question here about failures if the page is not already in DOM

Reply:
The failure would happen regardless, the refactor of this code did not change that. Navigation validation should be done outside of these functional methods, preferably inside the tests themselves, via assertions.